plugins/q/test/unit/q-brush.spec.js

```diff
@@ -199,6 +199,130 @@ describe('q-brush', () => {
     });
   });
 
+  describe('selectPivotCells', () => {
+    const layout = {
+      qHyperCube: {
```

Comment:
should move this into `beforeEach` to ensure pure object for each test

Reply:
Fixed!